app/src/main/java/com/example/fivenightsescape/monster/Monster.kt

```diff
@@ -0,0 +1,130 @@
+package com.example.fivenightsescape.monster
+
+import android.graphics.Color
+import android.location.Location
+import com.example.fivenightsescape.player.Player
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.model.Circle
+import com.google.android.gms.maps.model.CircleOptions
+import com.google.android.gms.maps.model.LatLng
+import kotlin.random.Random
+
+const val TYPE_STANDING = "standing"
+const val TYPE_MOVING = "moving"
+const val TYPE_WANDERING = "wandering"
+
+abstract class Monster(
+    val damage: Int = 1,
+    var player: Player,
+    var position: LatLng,
+    var mMap: GoogleMap)
```

Comment:
Property 'mMap' could be private.
In MapsActivity it's: `private lateinit var mMap: GoogleMap`